source/_posts/rails-docker.md

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
in content there should be link with previous content

Reply:
This is not linked to our previous posts. So can't add a link.